### PR TITLE
HOTFIX: EIP1-5198: EIP1-5200: Adds AED supporting format that accepts all values

### PR DESCRIPTION
**Pull request by vishalgupta-valtech**

### CODE REVIEW

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Print APIs
-  version: '1.12.3'
+  version: '1.13.0'
```
Comment (matt-wills-valtech) [inline src/main/resources/openapi/PrintAPIs.yaml:4]: Minor - why not 1.12.4?

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 2]
```diff
@@ -786,10 +786,20 @@ components:
         - dateTime
 
     SupportingInformationFormat:
+      title: SupportingInformationFormat
```
Comment (vishalgupta-valtech) [inline src/main/resources/openapi/PrintAPIs.yaml:789]: Can't extend this as VCA and other flows uses this, so had to add a new format enum

**Review by matt-wills-valtech — APPROVED** [review]

**Review by sentioservices — APPROVED** [review]
LGTM

File: src/main/resources/openapi/PrintAPIs.yaml (hunk 2 above)
Comment (Neil-Massey) [reply src/main/resources/openapi/PrintAPIs.yaml:789]: Should the temporary certificate not also support the other supporting information formats?

Comment (matt-wills-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:789]: Not sure I understand the question? @vishalgupta-valtech's change doesn't affect temp certs.

**Review by sentioservices — APPROVED** [review]
LGTM

File: src/main/resources/openapi/PrintAPIs.yaml (hunk 2 above)
Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:789]: @Neil-Massey I don't see temporary certs using `supportingInformationFormat` in spec before and also they are different to AEDs as AEDs needs to be generated (if requested) after actual removal from vca.

image

**Review by matt-wills-valtech — APPROVED** [review]
LGTM

File: src/main/resources/openapi/PrintAPIs.yaml (hunk 2 above)
Comment (Neil-Massey) [reply src/main/resources/openapi/PrintAPIs.yaml:789]: Sorry, missed two things, 1 this is a bug fix and 2 the endpoint I was thinking doesn't take a supporting document format

```
/eros/{eroId}/temporary-certificates/{gssCode}/explainer-document
```

**Pull request merged**